src/screens/Detail/BookedAndGrade.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { View, Text, Image, StyleSheet } from "react-native";
+import styled from "styled-components/native";
+import { FontAwesome } from "@expo/vector-icons";
+import { AntDesign } from "@expo/vector-icons";
+
+function BookedAndGrade(props) {
+  const { numberOfBooked, fripGrade } = props;
```

Comment:
함수형 컴포넌트에서는 이렇게 하시는 게 더 많이 쓰입니다 :)
```suggestion
function BookedAndGrade({ numberOfBooked, fripGrade }) {
```